lerobot/common/logger.py

```diff
@@ -95,8 +99,13 @@ def save_buffer(self, buffer, identifier):
         fp = self._buffer_dir / f"{str(identifier)}.pkl"
         buffer.save(fp)
         if self._wandb:
```

Comment:
This should have probably already been there
```suggestion
        if self._wandb and not self._disable_wandb_artifact:
```
cc @Cadene to confirm this is what we'd want